### ISSUE TITLE
Incorrect syntax of babel helpers

### ISSUE THREAD

**Issue by flying-sheep** [rollup/rollup-plugin-babel#33]
idk if this is a bug here or with babel-plugin-external-helpers, but i get this at the start of my file:

```js
(function () { 'use strict';

    var __commonjs_global = typeof window !== 'undefined' ? window : typeof global !== 'undefined' ? global : this;
    function __commonjs(fn, module) { return module = { exports: {} }, fn(module, module.exports, __commonjs_global), module.exports; }


    ...

    function babelHelpers_createClass () {
      function defineProperties(target, props) {
        for (var i = 0; i < props.length; i++) {
          var descriptor = props[i];
          descriptor.enumerable = descriptor.enumerable || false;
          descriptor.configurable = true;
          if ("value" in descriptor) descriptor.writable = true;
          Object.defineProperty(target, descriptor.key, descriptor);
        }
      }

      return function (Constructor, protoProps, staticProps) {
        if (protoProps) defineProperties(Constructor.prototype, protoProps);
        if (staticProps) defineProperties(Constructor, staticProps);
        return Constructor;
      };
    }();

    ...
```

the last line is apparrently invalid (says firefox). i assume you can’t invoke a function declaration like this?

**Comment by vannhi** [comment]
I have the same problem. The correct syntax should be
```javascript
    var babelHelpers_createClass = function() {
      function defineProperties(target, props) {
        for (var i = 0; i < props.length; i++) {
          var descriptor = props[i];
          descriptor.enumerable = descriptor.enumerable || false;
          descriptor.configurable = true;
          if ("value" in descriptor) descriptor.writable = true;
          Object.defineProperty(target, descriptor.key, descriptor);
        }
      }

      return function (Constructor, protoProps, staticProps) {
        if (protoProps) defineProperties(Constructor.prototype, protoProps);
        if (staticProps) defineProperties(Constructor, staticProps);
        return Constructor;
      };
    }();
```

**Comment by Victorystick** [comment]
@ericf has a fix for this in #31. Will test it before merging and releasing.

**Comment by flying-sheep** [comment]
fixed by #31
